src/Core/tests/DeviceTests/Handlers/HandlerTestBase.Tests.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.Maui.DeviceTests.Stubs;
+using Xunit;
+
+namespace Microsoft.Maui.DeviceTests
+{
+	public abstract partial class HandlerTestBase<THandler>
+	{
+		protected abstract Type GetStubType();
```

Comment:
Maybe this can be another generic arg for the test?

```csharp
public partial class EntryHandlerTests : HandlerTestBase<EntryHandler, EntryStub>
```

Then this can happen:

```csharp
public abstract partial class HandlerTestBase<THandler, TStub>
  where TStub : new(), IView
```

Or something like that.